### ISSUE TITLE
NSFW! kisshentai

### ISSUE THREAD

**Issue by gavbn** [jspenguin2017/uBlockProtector#255]
Before opening an issue, you MUST read the contribution guideline. Failure to do so may result in a ban. 

Test link: http://kisshentai.net/Hentai/Oni-Chichi-Refresh/Episode-004?id=2427

Screenshot: 
![capture](https://cloud.githubusercontent.com/assets/23123477/26688577/a92877a8-4710-11e7-8e26-041697cc2e8e.PNG)

Add everything else that you believe to be useful below: the site blocks ublock origin

**Comment by jspenguin2017** [comment]
You have not read the contribution guideline. This is a warning. Do not open another issue before reading it.

**Comment by ale5000-git** [comment]
@gavbn: The script doesn't work with Greasemonkey; you need [Violentmonkey](https://addons.mozilla.org/en-Us/firefox/addon/violentmonkey/) and the [script](https://jspenguin2017.github.io/uBlockProtector/#how).
